### PR TITLE
bug 1600173 updating self-provisioner clusterrole name

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1600173

@stuartchuan, will you PTAL?

### CODE REVIEW

**Comment by stuartchuan** [comment]
There were never `self-provisioners` cluster role in 3.9, I also checked 3.7 and 3.6 still don't have such cluster role.
```
# oc get clusterrole | grep self-provisioner
self-provisioner
# oc get clusterrolebinding | grep self-provisioner
self-provisioner                                                      /self-provisioner                                                                                                                                      management-infra/management-admin                                                  
self-provisioners                                                     /self-provisioner                                                                                       system:authenticated:oauth                                                                                                        
# openshift version
openshift v3.9.33
kubernetes v1.9.1+a0ce1bc657
etcd 3.2.16
```
Reviewed again 'Disabling Self-provisioning' part, I don't think we need this changes to apply. Only one confused point, will comment in line.

File: admin_guide/managing_projects.adoc [hunk 1]
```diff
@@ -116,18 +116,18 @@ Subjects:
   Group	system:authenticated:oauth
 ----
 
-. Remove the `self-provisioners`
+. Remove the `self-provisioner`
 xref:../architecture/additional_concepts/authorization.adoc#roles[cluster role binding]
 from the group `system:authenticated:oauth`.
```
Comment (stuartchuan) [inline admin_guide/managing_projects.adoc]: There is the only confused point, should changed to `3.Remove the self-provisioner cluster role from the group system:authenticated:oauth.`

File: admin_guide/manage_rbac.adoc [hunk 2]
```diff
@@ -403,7 +403,7 @@ Subjects:
   Group	system:unauthenticated
 
 
-Name:		self-provisioners
+Name:		self-provisioner
```
Comment (kalexand-rh) [inline admin_guide/manage_rbac.adoc]: @stuartchuan, based on the command output that you posted, this change is valid:
```
# oc get clusterrole | grep self-provisioner
self-provisioner
````

File: admin_guide/managing_projects.adoc (hunk 1 above)
Comment (kalexand-rh) [reply admin_guide/managing_projects.adoc]: Got it. Thank you @stuartchuan!

**Comment by stuartchuan** [comment]
Thank you for the updated, it's LGTM now.

**Comment by kalexand-rh** [comment]
I don't think this change needs peer review. Merging.

**Pull request merged**